### DISCUSSION TITLE
How to random sort/order with Fusejs

### DISCUSSION THREAD

**Discussion by afshinr1** [krisk/Fuse#551] [Q&A] [answered]
I want to randomly sort the results of the search, instead of by score. How can I do this? Ive been trying to use the SortFn option but can't seem to figure out exactly how to. Thanks!

**Comment by krisk** [comment, marked as answer]
Very late reply — sorry about that.

You can pass a custom `sortFn` that randomizes the order:

```js
const fuse = new Fuse(list, {
  keys: ['title'],
  sortFn: () => Math.random() - 0.5
})
```

Or just shuffle the results after the search:

```js
const results = fuse.search('query')
results.sort(() => Math.random() - 0.5)
```
